nbconvert/filters/markdown_mistune.py

```diff
@@ -28,6 +31,10 @@
 from nbconvert.filters.strings import add_anchor
 
 
+class InvalidNotebook(Exception):
```

Comment:
This un unrelated to the PR itself, `InvalidNotebook` was used in this file but never imported. Importing it was resulting in a circular import so we need to define it here (it's only used in this file anyway)